### DISCUSSION TITLE
wp-admin access denied

### DISCUSSION THREAD

**Discussion by iktomi3** [code4recovery/12-step-meeting-list#601] [Q&A] [answered]
After upgrading the 12SML plugin in Wordpress, access to the admin features is denied: results in the "white screen of death" (as I've heard it referenced). Disabling the plugin via Cpanel solved the access issue, but even rolling back the plugin produces no results for the Meeting Guide/List/plugin. Any thoughts? Or better yet, directions on how to correct the issue? They're a small group without a lot of technical experience and I'm trying to help them out, but I too am stumped. Thank you.

**Comment by tech2serve** [comment]
> After upgrading the 12SML plugin in Wordpress, access to the admin features is denied: results in the "white screen of death" (as I've heard it referenced). Disabling the plugin via Cpanel solved the access issue, but even rolling back the plugin produces no results for the Meeting Guide/List/plugin. Any thoughts? Or better yet, directions on how to correct the issue? They're a small group without a lot of technical experience and I'm trying to help them out, but I too am stumped. Thank you.

TBH, this is not something that I've seen as an issue with TSML before. And hundreds of sites have updated to 3.12.1 and your question is the only one brought to our attention. This causes me to suspect something with your site rather than the plugin itself.

I personally have very little knowledge about how to handle a "white screen of death" other than searching the Internet for idea.

[WP Beginner](https://www.wpbeginner.com/wp-tutorials/how-to-fix-the-wordpress-white-screen-of-death/) has an article that suggests increasing the memory limit of the website, and disabling other plugins, etc. Please try working through this list of ideas, and let us know if you regain control of your site.

What is the URL of the site?

**Reply by iktomi3** [reply]
Awesome that you took the time to reply, thank you. This is for  indyaa.org
I'll get to digging through the unusual suspects - we've tried much of what has been suggested. Removing TSML solved admin access. Rolling it back produces no admin access issues, but then doesn't display the meetings.
We'll keep at it and happily report back when we find a solution.

**Reply by Grateful-Viv** [reply]
I would also switch to a stable Wordpress theme, just to rule that out. Latest WP is v5.8.2. Try it with 2019 theme. 

Do they have a backup prior to updating the TSML plugin? If so, please restore and then export the meeting data which will give you a csv file.

**Reply by Grateful-Viv** [reply]
Also look here for some suggestions https://wordpress.org/support/article/common-wordpress-errors/

**Reply by 97eighty** [reply]
@iktomi3 Have you turned on debugging to see if there is a PHP error thrown?

**Comment by brianw-area24** [comment, marked as answer]
I'm guessing this is another case where the web site is running php 8. If that's the case, then the latest version of the plugin, 3.12.2 should fix this issue.

**Reply by tech2serve** [reply]
@iktomi3,

Are you running PHP 8? And did upgrading to 3.12.2 fix your problem?

Feedback will help us close this Q&A and provide better support. 

Thank you!
